larray/util/misc.py

```diff
@@ -131,8 +136,9 @@ def get_min_width(table, index):
     return max(longest_word(row[index]) for row in table)
 
 
+# XXX: what fullinfo is used for and when ?
```

Comment:
The ``fullinfo`` was already there in the "initial commit" =D

Reply:
hehe 😄 This is a leftover from the code copied from LIAM2 then. Now that I think of LIAM2, I think this was because this code is used when outputting to .csv files in LIAM2 and we want all the precision we can get in that case.